service/gcs/runtime/runc/runc.go

```diff
@@ -324,13 +325,20 @@ func (c *container) GetAllProcesses() ([]runtime.ContainerProcessState, error) {
 	if err != nil {
 		return nil, err
 	}
+	l := logrus.WithField("cid", c.id)
+	l.WithField("pids", runningPids).Debug("running container pids")
 
 	pidMap := map[int]*runtime.ContainerProcessState{}
 	// Initialize all processes with a pid and command, leaving CreatedByRuntime
 	// and IsZombie at the default value of false.
 	for _, pid := range runningPids {
 		command, err := c.r.getProcessCommand(pid)
 		if err != nil {
+			if errors.Is(err, unix.ENOENT) {
```

Comment:
I think we should add the same check in the `GetRunningProcesses` function that is just above this function?